Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/xenial64"
+
+  config.vm.provision "shell", inline: <<-SHELL
+
+    echo "Provisioning Start"
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get -qq update && apt-get -qq dist-upgrade
+
+    echo "Installing MySql"
+    sudo -E apt-get -qq -y install mysql-server
+    /etc/init.d/mysql stop
+    mysqld --skip-grant-tables &
+    mysql -u root -e "UPDATE user SET Password=PASSWORD('pass123') WHERE User='root'; FLUSH PRIVILEGES; exit;"
```

Comment:
I don't think you'd want the interactive install to come up while you're provisioning a vagrant box anyways. It'd be nice if the provisioner could ask some questions, but I don't think Vagrant really supports that in a standard way. 

People could always change the password, but I have a feeling that no one cares what their dev db password is.